pkg/cmd/subcommand/uninstall.go

```diff
@@ -169,49 +174,63 @@ func (i *Uninstall) RunUninstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 		return nil
 	}
 
-	grp, gctx := errgroup.WithContext(ctx)
-	defer gctx.Done()
+	grp, ctx := errgroup.WithContext(ctx)
```

Comment:
not `WithTimeout` like the install part?

Reply:
fixed